### ISSUE TITLE
A new factory

### ISSUE THREAD

**Issue by SuperThunder958** [Anuken/Mindustry-Suggestions#3277]
### Describe the content or mechanics you are proposing.

How about  a who new type of production block called the resource Crafter where you can turn certain resources into other resources. And you unlock it by capturing at least 3 sectors and 1 enemy base. I'm going to list a couple examples below with some symbols ([ * = could be extremely useful ] [ ? = Maybe not useful] [ ^ = useful depending on little to none of a certain resource] [•• = optional] )

Graphite= spore pod + sand
? Copper = lead + pryrite 
^ Titanium =  copper + graphite + lead + and/or spore pod
^ Scrap = sand + lead
Spore pod = coal + pryrite +  bit of water
? ^ Lead = scrap + spore pod + slag  -or- oil 
Surge alloy =  silicon + plasiamn 
Plasiamn = Titanium + spore pod
Silicon = sand + coal -or- pryrite
* ^ ? Thorem = lead + spore pod + sand + Titanium
Phase frabic = sand + thorem + silicon

And so on but that's just a little idea so that away the separator and disassember has a rival but so it truly is they rival there should be a teir 2 of it Where you are able to craft thorem, phase frabic, plasiamn, surge alloy.

I hope you guys like my idea and sorry bout my typos I was trying as hard as I can to spell while rushing at the same time cause my phone was under 10% while typing this.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

Acreage to fix them outages of resources say if there's not enough of one resource on one sector they can use that to make that resource and vice versa

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by RealYeet** [comment]
Does it need to be explained why this is a bad idea?

**Comment by itcannotbe** [comment]
Ah that's why this sounds similar....

https://github.com/Anuken/Mindustry-Suggestions/issues/3261#issuecomment-1001664520

**Comment by random-guy1285** [comment]
This doesn't fit mindustry nor is even useful.
Spore pod + sand = coal? Just put the pod in a spore press and the oil in a coal centrifuge.
lead + pyratite = copper.  What sector doesn't have copper.
Copper + lead + graphite + and/or spore pod = titanium. The hell do you mean and/or.  We can already get titanium in every place that matters. If not, launch pads.
Sand + lead = scrap. Why the actual hell would you want scrap.
Coal + pyratite + water = spore pod. JUST USE A HARVESTER.
Scrap + spore pod + slag/oil = lead. You can already get lead from scrap/slag.
Silicon + plastanium = surge alloy. How does this even make sense.
Titanium + spore pod = plastanium. spore press.
sand + coal/pyratite = silicon. Silicon crucible called, it wants its recipe back.
Lead + spore pod + sand + titanium = thorium. I'm going to have a aneurysm.
Sand + thorium + silicon = phase fabric. Just use a phase weaver.

In conclusion this idea is outclassed by existing blocks.
"but the sector doesn't have ____"  launch pads existing for a reason people.
PS. stop trying to add spore pods into every recipe.

**Comment by EclipseTheOldOne** [comment]
~~that shit sounded like akchemisting~~

**Comment by Ymabert** [comment]
it is not entirely a bad idea, it would be nice to be able to create (as in your example) / disassemble all the materials with a special tool, but it would be illogical to create something for example with glass and generate copper output for example, or any material, because it should generate waste, for example if silicon enters with lead it generates waste 1 coal and 1 of sand as discarded material and glass, it would be nice to have a choice based on what enters, however it is a disassembly of the incoming material would make more sense, but it would be useless, because then there is scrap if you need to have materials of all kinds;  having a factory to create materials using other manufactured materials is useless

**Comment by Falling10fruit** [comment]
We don't need Phase to become more expensive

**Comment by EclipseTheOldOne** [comment]
how in de fuck you turn copper, lead and graphite into titanium

**Comment by ChivalricMender** [comment]
This might already be planned. Anuke did tease an alternative silicon smelter using graphite instead of coal.

**Comment by EclipseTheOldOne** [comment]
it still fucking make sense because it's *not* akchemist, silicon making process is processing sand silic with cacbon (with contained in graphite or coal) to make silicon, and not turning other material into a different thing

> This might already be planned. Anuke did tease an alternative silicon smelter using graphite instead of coal.
